examples/server/server.cpp

```diff
@@ -676,6 +676,25 @@ struct llama_server_context
             }
         }
 
+        const auto &samplers_sequence = data.find("samplers");
+        if (samplers_sequence != data.end() && samplers_sequence->is_array())
+        {
+            std::vector<std::string> sampler_names;
+            for (const auto &sampler_name : *samplers_sequence)
+            {
+                if (sampler_name.is_string())
+                {
+                    sampler_names.emplace_back(sampler_name);
+                }
+            }
+            auto sampler_types = sampler_types_from_names(sampler_names, false);
+            slot->sparams.samplers_sequence.swap(sampler_types);
```

Comment:
Same thing as below:

slot->sparams.samplers_sequence = sampler_types_from_names(sampler_names, false);

Reply:
Fixed.

Reply:
Btw, force-pushing makes a bit more difficult to review the changes. Small commits in the PRs are fine